### PR TITLE
Added Vue js docs and file handling with azure storage example

### PR DESCRIPTION
**Pull request by juicycleff**

### CODE REVIEW

**Comment by marktani** [comment]
I'll review the example shortly.
I'll leave reviewing the quickstart to @nikolasburk 🙂

File: examples/file-handling-azure-storage/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "dotenv -- nodemon -x ts-node -e ts,graphql src/index.ts",
+    "debug": "dotenv -- nodemon -x 'ts-node --inspect' -e ts,graphql src/index.ts",
+    "playground": "dotenv -- graphql playground",
+    "bind": "graphql prepare",
+    "build": "rm -rf dist && tsc"
+  },
+  "dependencies": {
```
Comment (marktani) [inline examples/file-handling-azure-storage/package.json]: * please use latest versions for all dependencies here, most importantly the `graphql` and `prisma` related ones 🙂 
* the same for the `graphql-cli` and `prisma` dev dependencies

File: examples/file-handling-azure-storage/README.md [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+Next, navigate into the downloaded folder and install the NPM dependencies:
+
+```sh
+cd file-handling-azure-storage
+yarn install
+```
+
+### 2. Setup your Azure storagee container & set environment variables
+
+The S3 bucket will be the storage container for your files. Here is how you can create one:
```
Comment (marktani) [inline examples/file-handling-azure-storage/README.md]: It's not a S3 bucket (not familiar with the Azure terminology)

File: examples/file-handling-azure-storage/README.md [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+
+1. Navigate to the [Azure Portal](https://portal.azure.com) in your browser.
+1. When you're there, click **Create a new resource**.
+    ![image_1] (https://www.dropbox.com/s/e6yf9wyyepd0wzb/first.PNG?dl=0)
```
Comment (marktani) [inline examples/file-handling-azure-storage/README.md]: please use imgur instead

File: examples/file-handling-azure-storage/README.md [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+    ![image_3](https://www.dropbox.com/s/3okrlm8oga3qimm/5.PNG?dl=0)
+5. After getting the keys click on **overview**
+    ![image_3](https://www.dropbox.com/s/7tnwrpj7b738cd2/4.PNG?dl=0)
+5. Then click on **blob** to create a new container
+6. Create a container and keep the name of your container
+1. go to `src/modules/fileApi.ts` and change the container variable to the name of your container e.g. `const container = 'containerName';`
+
+Next, you need to copy the values for the **Access key ID** and **Secret access key** into your GraphQL server project. They're _used_ in [`index.ts`](./src/index.ts), but referenced as environment variables which are defined in [`.env`](./.env).
+
+For each environment variable in `.env` that's prefixed with `S3`, set the corresponding value by replacing the placeholder.
```
Comment (marktani) [inline examples/file-handling-azure-storage/README.md]: S3

**Comment by nikolasburk** [comment]
Hey @juicycleff, we're currently cleaning up our PRs. Apologies for not having looked at it in detail earlier, unfortunately it is outdated by now so I'll close it. Please keep opening PRs in the future if you see any issues with our docs 🙏

**Pull request closed without merging**